### DISCUSSION TITLE
Problemi di accesso da IP pubblico (esterno)

### DISCUSSION THREAD

**Discussion by alecrespi78** [globaleaks/globaleaks-whistleblowing-software#3895] [Q&A] [answered]
Buongiorno a tutti,
ho installato con successo Globaleaks (4.13.22) su un server Ubuntu 22.04.3 LTS.
La piattaforma funziona correttamente se ci si connette tramite la rete aziendale (192.168.x.x) sulla porta 80.
La piattaforma funziona correttamente se ci si connette tramite TOR (con il link onion).
Ho dei problemi a raggiungere la piattaforma da un indirizzo IP pubblico.
Ci sono delle porte particolari che devono essere reindirizzate (dagli altri thread mi sembra di capire la 80 e la 443)?

Una volta assegnato l'indirizzo IP pubblico ho mappato sul firewall la porta X (esterna) con la 80 interna ma senza risultati.
Sul firewall sono mappate altre porti su altre destinazioni e tutto funziona correttamente.

Non riesco a capire dove sia l'errore.
Spero che qualcuno possa aiutarmi.

Grazie in anticipo.
Alessandro

**Comment by evilaliv3** [comment]
Può fornirci il dominio o l'IP di riferimento?

**Reply by alecrespi78** [reply]
https://whistleblowing.fmbgroup.it/
195.210.40.226

Grazie

**Comment by spomar** [comment, marked as answer]
il servizio globaleaks solo sugli ip indicati a termine installazione e non c'e' la tua porta wan (che potrebbe variare)
Ti conviene proxare verso il servizio globaleaks con un webserver che metterai su porta wan
